clients/java/src/main/java/io/camunda/client/api/search/filter/ElementInstanceFilter.java

```diff
@@ -63,6 +63,14 @@ public interface ElementInstanceFilter extends SearchRequestFilter {
    */
   ElementInstanceFilter elementId(final String value);
 
+  /**
+   * Filters element instances by element name.
```

Comment:
🔧 We should note here that this only works for data created with 8.8 and onwards. Instances from prior versions don't contain this data.